robot/rospackages/src/image_scp/image_scp.py

```diff
@@ -0,0 +1,79 @@
+import fnmatch
+import os
+from paramiko import SSHClient
+from scp import SCPClient, SCPException
+import argparse
+from config import (host, user, remote_path_1)
+"""
+Script takes in a directory name as a command line argument. This directory should contain
+a config.txt file and any images that need to be transered.
+SSH keys need to be configured on both remote and local machine in order for the script to work.
+Host name, user name, and remote path are all imported from config.txt file. Update the information
+there so that this script is portable.
```

Comment:
```suggestion
Script takes in a directory name as a command line argument. This directory should contain
a config.txt file and any images that need to be transfered.
SSH keys need to be configured on both remote and local machines in order for the script to work.
Host name, user name, and remote path are all imported from config.txt file. Update the information
there so that this script is portable.
```